.gitignore

```diff
@@ -42,3 +42,4 @@ example/package-lock.json
 
 # Storybook
 storybook-static
+.jest-test-results.json
```

Comment:
This stores the results of the jest tests and is used to display them in the `tests` tab in the storybook ui.